ansible/roles/eos/templates/t1-64-lag-tor.j2

```diff
@@ -100,6 +100,9 @@ interface {{ bp_ifname }}
 router bgp {{ host['bgp']['asn'] }}
  router-id {{ host['interfaces']['Loopback0']['ipv4'] |  ipaddr('address') }}
  !
+ graceful-restart restart-time {{ bgp_gr_timer }}
```

Comment:
where is this timer is defined?

Reply:
Good catch, thanks!
This bgp_gr_timer is used when deploy VMs for testing. It is defined in sonic-mgmt/ansible/group_vars/eos/eos.yml. I missed this file in the PR. I will add it in new commits.